src/main/Main.cpp

```diff
@@ -160,6 +160,8 @@ start(io_context& ioc, std::uint32_t numThreads)
         v.emplace_back([&ioc] { ioc.run(); });
 
     ioc.run();
+    for (auto& t : v)
```

Comment:
This might be unnecessary but i'm not opposed to adding it just for completeness sake. 
I hope `jthread` will be available soon.